arangod/RocksDBEngine/RocksDBEdgeIndex.cpp

```diff
@@ -890,6 +892,9 @@ void RocksDBEdgeIndex::handleValNode(VPackBuilder* keys,
 
 void RocksDBEdgeIndex::afterTruncate(TRI_voc_tick_t tick,
                                      arangodb::transaction::Methods* trx) {
+  if (unique()) {
```

Comment:
I think the edge Index can never be unique, so this code in fact is unreachable.
But it would be correct if reachable :+1:

Reply:
I agree. Will put a code comment there after merging!